src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -16,7 +16,7 @@
 import cherrypy
 
 
-def ApiController(path):
+def ApiController(path=''):
```

Comment:
I don't understand what is the purpose of this change. If path is an empty string, then what will be the final URL for the controller?

Reply:
That depends. If it's the first controller attached, a path makes sense, for any nested controller, it won't be used anymore.

Reply:
@p-na "Nested controllers" don't need to be decorated with `@ApiController`

Reply:
Revised

Reply:
please revert this change, since is not needed anymore